x-pack/plugins/security_solution/tsconfig.json

```diff
@@ -185,6 +185,7 @@
     "@kbn/core-http-common",
     "@kbn/search-errors",
     "@kbn/stack-connectors-plugin",
-    "@kbn/elastic-assistant-common"
+    "@kbn/elastic-assistant-common",
+    "@kbn/lens-embeddable-utils"
```

Comment:
This is the only change I've seen in this PR that isn't directly related to code organization, curious what this change does?

Reply:
Nice catch!

It was autogenerated by kibanamachine bot.
https://github.com/elastic/kibana/pull/173499/commits/2d80600271b6ddb16cd09b712a753d3c364eb47e